inference/server/alembic/versions/2023_05_07_0923-19178d144265_add_sibling_active_to_message.py

```diff
@@ -0,0 +1,28 @@
+"""add_sibling_active_to_message
```

Comment:
Can we merge both Alembic scripts into one before merging this? Will help reduce clutter